package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "scratchr2_translations",
+  "version": "1.0.0",
+  "description": "scratchr2_translations ======================",
```

Comment:
Suggestion: `"scratchr2_translations ====================== true"`. Just to be extra sure.